images-list

```diff
@@ -317,6 +319,7 @@ quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.9.4
 quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.9.6
 quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.9.8
 quay.io/cilium/startup-script rancher/mirrored-cilium-startup-script 62bfbe88c17778aad7bef9fa57ff9e2d4a9ba0d8
+quay.io/coreos/flannel rancher/mirrored-coreos-flannel v0.14.0
```

Comment:
Do we use upstream flannel? I think `rancher/coreos-flannel:v0.13.0-rancher1` was used in previous releases.

Reply:
v0.12.0 (previous versions) didnt have manifest support, this one has.

Reply:
good :+1: